docs/antora/modules/ROOT/pages/Library_Dependencies.adoc

```diff
@@ -172,10 +172,43 @@ To render a tree of dependencies (transitive included) and possible upgrades for
 │  │  │  └─ com.lihaoyi:sourcecode_2.13:0.2.1 -> 0.3.0 (possible incompatibility)
 ----
 
-You can also compile your module(s) and examine files such as `ivyDeps.json` and `transitiveIvyDeps.json` in your `out` build's folder for a given module. `ivyDepsTree.log` contains file output of the above ivyDepsTree command too.
+After compiling your module(s) you can find and examine files such as `ivyDeps.json` and `transitiveIvyDeps.json` in your `out` build's folder for a given module.
+After running the `ivyDepsTree` command you'll also the `ivyDepsTree.json` and `ivyDepsTree.log` file that contain the output of the above ivyDepsTree command.
```

Comment:
It's hard to read to me. Is there a verb missing? "you'll have ..." or "you can find ..."?

Reply:
ah sorry, yea I forgot the `find`.